### ISSUE TITLE
System Messages utility doesn’t use the correct language

### ISSUE THREAD

**Issue by brandonkelly** [craftcms/cms#7992]
I have one site defined with its language set to German. My admin account is set to English. When I go to Utilities -> System Messages I can only edit the messages in English. If I want to change the System Messages in German, I have to switch my account preference to German.

_Originally posted by @schwarzdesign in https://github.com/craftcms/cms/issues/7201#issuecomment-848581589_

**Comment by brandonkelly** [comment]
Craft 3.6.16 is out now with that fix.

**Comment by DavidKabelitz** [comment]
@brandonkelly i have a question on this topic. We also have a site with both languages and two sign ups form for user. But all messages that send (account activation) are in German. How we can control that when someone signs up at the english page just got english emails?

**Comment by brandonkelly** [comment]
@DavidKabelitz Messages will be sent in the language chosen by the currently requested site, at the time the email is sent. However Craft doesn’t keep track of which site the user originally signed up from, so there’s no way to force them to receive emails from that original language.

**Comment by DavidKabelitz** [comment]
@brandonkelly yeah it really works like a charm with all form. Just our sign up form was form using a plugin but this is also solved already. thanks
